### PR TITLE
feat: Added GraphQL in backend database Category

### PR DESCRIPTION
**Pull request by NarendraKoya999**
## Changes proposed

- **Added GraphQL Details in database sub category in backend Category**

### CODE REVIEW

File: database/backend/database.json [hunk 1]
```diff
@@ -61,5 +61,12 @@
     "url": "https://docs.aws.amazon.com/amplify/",
     "category": "backend",
     "subcategory": "database"
+  },
+  {
+    "name": "GraphQL",
+    "description": "GraphQL provides a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they need and nothing more, makes it easier to evolve APIs over time, and enables powerful developer tools.",
```
Comment (CBID2) [inline database/backend/database.json:67]:
```suggestion
    "description": "GraphQL provides a comprehensive description of the data in your API, which gives clients the power to ask for exactly what they need and nothing more and make it easier to evolve APIs over time. It also enables powerful developer tools.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Made a tweak to your description

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**